extensions/arc/src/controller/generated/dusky/api/databaseRouterApi.ts

```diff
@@ -0,0 +1,2647 @@
+    public async createDuskyDatabase (ns: string, serviceName: string, duskyObjectModelsDatabase?: DuskyObjectModelsDatabase, options: {headers: {[name: string]: string}} = {headers: {}}) : Promise<{ response: http.IncomingMessage; body: DuskyObjectModelsDatabase;  }> {
+        const localVarPath = this.basePath + '/dusky/databases/{ns}/{serviceName}/databases'
+            .replace('{' + 'ns' + '}', encodeURIComponent(String(ns)))
+            .replace('{' + 'serviceName' + '}', encodeURIComponent(String(serviceName)));
+        let localVarQueryParameters: any = {};
+        let localVarHeaderParams: any = (<any>Object).assign({}, this._defaultHeaders);
+        const produces = ['application/json'];
+        // give precedence to 'application/json'
+        if (produces.indexOf('application/json') >= 0) {
+            localVarHeaderParams.Accept = 'application/json';
+        } else {
+            localVarHeaderParams.Accept = produces.join(',');
+        }
+        let localVarFormParams: any = {};
+
+        // verify required parameter 'ns' is not null or undefined
+        if (ns === null || ns === undefined) {
+            throw new Error('Required parameter ns was null or undefined when calling createDuskyDatabase.');
+        }
+
+        // verify required parameter 'serviceName' is not null or undefined
+        if (serviceName === null || serviceName === undefined) {
+            throw new Error('Required parameter serviceName was null or undefined when calling createDuskyDatabase.');
+        }
+
+        (<any>Object).assign(localVarHeaderParams, options.headers);
+
+        let localVarUseFormData = false;
+
+        let localVarRequestOptions: localVarRequest.Options = {
+            method: 'POST',
+            qs: localVarQueryParameters,
+            headers: localVarHeaderParams,
+            uri: localVarPath,
+            useQuerystring: this._useQuerystring,
+            json: true,
+            body: ObjectSerializer.serialize(duskyObjectModelsDatabase, "DuskyObjectModelsDatabase")
+        };
+
+        let authenticationPromise = Promise.resolve();
+        if (this.authentications.BasicAuth.username && this.authentications.BasicAuth.password) {
+            authenticationPromise = authenticationPromise.then(() => this.authentications.BasicAuth.applyToRequest(localVarRequestOptions));
+        }
+        if (this.authentications.BearerAuth.accessToken) {
+            authenticationPromise = authenticationPromise.then(() => this.authentications.BearerAuth.applyToRequest(localVarRequestOptions));
+        }
+        authenticationPromise = authenticationPromise.then(() => this.authentications.default.applyToRequest(localVarRequestOptions));
+
+        let interceptorPromise = authenticationPromise;
+        for (const interceptor of this.interceptors) {
+            interceptorPromise = interceptorPromise.then(() => interceptor(localVarRequestOptions));
+        }
+
+        return interceptorPromise.then(() => {
```

Comment:
You're in async/await, why do this?

Reply:
FYI I'm not blocking on this, just noticed it.

Reply:
Yeah so this is all generated code so isn't stuff we wrote (although if we do see any actual bugs we should fix them for sure)